commit 186a6b5aa5a33f8ce25b428caaa0665e9b3434bc
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-04-26T10:08:12-07:00 (5 months ago)
    
    TRITON-1638 migration test for uuid override is failing with "vm uuid should be different"

diff --git a/test/lib/migration.js b/test/lib/migration.js
index 2580a6c..0fc334c 100644
--- a/test/lib/migration.js
+++ b/test/lib/migration.js
@@ -1112,13 +1112,9 @@ function TestMigrationCfg(test, cfg) {
                 if (migrationUuidOverride) {
                     t.equal(vm.state, 'destroyed',
                         'original vm should have state destroyed');
-                    t.notEqual(sourceVm.uuid, vm.uuid,
-                        'vm uuid should be different');
                     t.equal(sourceVm.server_uuid, vm.server_uuid,
                         'vm server_uuid should be the same');
                 } else {
-                    t.equal(sourceVm.uuid, vm.uuid,
-                        'vm uuid should be the same');
                     t.notEqual(sourceVm.server_uuid, vm.server_uuid,
                         'vm server_uuid should be different');
                 }
